src/style.css

```
@@ -0,0 +1,98 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+
+  color: white;
+  font-size: 100px;
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
```

You can delete flex-direction because row is default value for flex-direction

```suggestion
.container {
  display: flex;
```